src/ButtonGroup.js

```diff
@@ -25,9 +25,13 @@ ButtonGroup.prototype.makeButton = function(ctx, btn) {
     var downAlpha = btn.downAlpha;
     var upAlpha = btn.upAlpha;
     var overAlpha = btn.overAlpha;
+    var active = btn.active;
+    var faded = btn.faded;
     if (downAlpha == null) {downAlpha = 0.8;}
     if (upAlpha == null) {upAlpha = 1;}
     if (overAlpha == null) {overAlpha = 0.5;}
+    if (active == null) {active = false;}
+    if (faded == null) {faded = false;}
```

Comment:
This is a mistake we didn't see previously but I think all these should be `===` instead of `==`